specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -41,15 +41,17 @@ paths:
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/expand"
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
         - in: query
-          name: kadastralegemeente
+          name: kadastralegemeente__code
+          description: "De code van de kadastrale gemeente, deel van de kadastrale aanduiding van de onroerende zaak. De doe en waarde zijn vastgelegd in een Waardelijst."
           schema: 
             type: "string"  
+            example: "50"
         - in: query
           name: perceelnummer
           description: "Het perceelnummer dat een geheel perceel of een complex uniek\
               \ identificeert binnen de sectie. Per kadastrale gemeente en per sectie\
               \ heeft een perceel een perceelnummer oplopend door de jaren heen van\
-              \ `00001` tot max `99999`"
+              \ 1 tot max 99999"
           schema:
             type: "integer"
             maximum: 99999
```

Comment:
waarom dan niet ook een minimum? Dan kan dat uit de description en is het technisch geborgd.

Reply:
Minimum toevoegen is geen probleem. In de description wordt ook uitgelegd dat het "door de jaren oploopt van 1 tot maximaal 99999." Ik kan niet inschatten of dat dan irrelevant wordt, dus lijkt mij dat dat stukje description gewoon blijft staan.